docs/rc.xml.all

```diff
@@ -12,6 +12,7 @@
     <gap>0</gap>
     <adaptiveSync>no</adaptiveSync>
     <reuseOutputMode>no</reuseOutputMode>
+    <cycleViewOSD>yes</cycleViewOSD>
     <cycleViewPreview>no</cycleViewPreview>
     <cycleViewOutlines>yes</cycleViewOutlines>
```

Comment:
Honestly I am not a big fan of the naming of these 3 config options but I can't  up with a better name either and breaking user config is also really bad. So even if we come up with a better name we should likely still accept the old ones as alternatives.

For the new config option, maybe we could use `<cycleView>` for now?
I'd say not every user can be assumed to know what `OSD` means.

But that could then potentially be thought of disabling all of the `A-Tab` switching, including previews and outlines.
Soo... not sure what to do here. Thoughts?

CC @johanmalm @jlindgren90